tests/processes/ProcessingTest.py

```diff
@@ -98,13 +83,12 @@ def setUp(self):
 
     def test_run(self):
         self.sections['cli'].append(Setting('jobs', '1'))
-        cache = FileCache(self.log_printer, 'coala_test', flush_cache=True)
+        cache = FileCache('coala_test', flush_cache=True)
         results = execute_section(self.sections['cli'],
                                   self.global_bears['cli'],
                                   self.local_bears['cli'],
-                                  lambda *args: self.result_queue.put(args[2]),
+                                  lambda *args: self.result_queue.put(args[1]),
```

Comment:
In args list, log_printer was present, can't recall whether on 0th position or 1st. After removing it, indices for trailing elements has to be reduced by 1.